### ISSUE TITLE
Installing globally throws "not found" error

### ISSUE THREAD

**Issue by jmiskinis** [gajus/eslint-plugin-jsdoc#795]
## Expected behavior
I use [the Meson build system](https://mesonbuild.com/) to build my app and perform tests via GitLab CI. [According to the `eslint-plugin-jsdoc` documentation](https://github.com/gajus/eslint-plugin-jsdoc#installation), tools like `meson` should be able to access and use global `eslint` and `eslint-plugin-jsdoc` installations.
## Actual behavior
Testing with global `eslint` installation throws an error:
```
ESLint couldn't find the plugin "eslint-plugin-jsdoc".

(The package "eslint-plugin-jsdoc" was not found when loaded as a Node module from the directory "/home/user/...".)
```
## ESLint Config

https://gitlab.com/jmiskinis/gnome-shell-extension-task-widget/-/blob/master/.eslintrc.yml
https://gitlab.com/jmiskinis/gnome-shell-extension-task-widget/-/blob/master/lint/eslintrc-gjs.yml
https://gitlab.com/jmiskinis/gnome-shell-extension-task-widget/-/blob/master/lint/eslintrc-overrides.yml

## .gitlab-ci.yml
```yaml
default:
  image: ubuntu:latest

stages:
  - build
  - release

build:
  stage: build
  before_script:
    - export DEBIAN_FRONTEND=noninteractive
    - apt-get -qq update
    - > 
      apt-get -qq install -y python3-pip ninja-build libglib2.0-dev
      libxml2-utils zip gettext
    - pip3 install meson
    - curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
    - export NVM_DIR="$HOME/.nvm" && . "$NVM_DIR/nvm.sh"
    - nvm install node
    - npm install -g eslint@7 eslint-plugin-jsdoc
  script:
    - meson _build -Dper_user=true -Dbuild_zip=true
    - meson test -C _build
[...]
```
## Environment

- Node version: 16.11.1
- ESLint version: eslint@7
- `eslint-plugin-jsdoc` version: latest (36.1.1?)

Am I doing something wrong here? ESLint tests without the use of `eslint-plugin-jsdoc` in ESLint config file run fine.

**Comment by brettz9** [comment]
I'm afraid I don't know. Could be a general ESLint bug. Have you tried it with ESLint 8? It seems I might have seen a regression like this at one point where older ESLint started having problems finding plugins.

I'd also recommend the latest `eslint-plugin-jsdoc`?

**Comment by jmiskinis** [comment]
The problem persists with ESLint 8. However, I was previously unaware of ESLint's `--resolve-plugins-relative-to` flag. Using it with ESLint 8 and in combination with `npm root -g` solves the issue for me:

```meson
eslint = find_program('eslint', required: false)
plugins = run_command('npm', 'root', '-g', check: true)

if eslint.found()
  test('ESLint', eslint, args: [join_paths(meson.project_source_root() , 'src'),
    '--resolve-plugins-relative-to=' + plugins.stdout().strip() + '/'])
endif
```
